packages/block-editor/src/components/block-edit/index.js

```diff
@@ -49,6 +53,7 @@ export default function BlockEdit( {
 	const layoutSupport =
 		hasBlockSupport( name, 'layout', false ) ||
 		hasBlockSupport( name, '__experimentalLayout', false );
+	const originalBlockClientId = useOriginalBlockOnlyUseOnce( clientId );
```

Comment:
Not sure I like this, we are adding another block-editor subscription. Could you move this to the private context provider where we select all the needed data for a block?

Reply:
We can :) just noting that we were already doing this though.

Reply:
Let me know if I did it properly.